CoreNotesView/src/au/gov/asd/tac/constellation/views/notes/NotesViewTopComponent.java

```diff
@@ -53,85 +54,98 @@
         notesViewPane = new NotesViewPane(notesViewController);
-        
+
         initContent();
     }
-    
+
     @Override
     protected void handleNewGraph(final Graph graph) {
-
         if (needsUpdate() && graph != null) {
             notesViewPane.selectAllFilters();
-            notesViewPane.clearNotes(false);
-            notesViewPane.prepareNotesViewPane(notesViewController);
+            notesViewPane.clearAllNotes();
+            notesViewController.readState(graph);
         }
     }
-    
+
     @Override
     protected void handleGraphClosed(final Graph graph) {
-
         if (needsUpdate() && graph != null) {
             notesViewPane.closeEdit();
-            notesViewPane.clearNotes(false);
-            notesViewPane.prepareNotesViewPane(notesViewController);
+            notesViewPane.clearAllNotes();
         }
     }
 
     @Override
     protected void handleComponentOpened() {
-        // Listener is not removed so that plugin reports created when the Notes View is not open will render when it is opened later.
+        /**
+         * listener is not removed so that plugin reports created when the Notes
+         * View is not open will render when it is opened later.
+         */
         GraphReportManager.addGraphReportListener(this);
     }
 
     @Override
     protected void handleComponentClosed() {
-        
         notesViewPane.closeEdit();
         notesViewPane.selectAllFilters();
-        notesViewPane.clearNotes(true);
+        notesViewPane.clearAllNotes();
     }
-    
+
     @Override
     protected NotesViewPane createContent() {
         return notesViewPane;
     }
-    
+
     @Override
     protected String createStyle() {
-        return "resources/notes-view.css";
+        return null;
     }
-    
+
     /**
      * Triggers when plugin reports are added or removed.
-     * 
+     *
      * @param pluginReport
      */
     @Override
     public void newPluginReport(final PluginReport pluginReport) {
-        // Omit plugin reports from the Notes View and Quality Control View.
-        if ((!pluginReport.getPluginName().contains("Notes View"))
-                && (!pluginReport.getPluginName().contains("Quality Control View"))) {
-            notesViewPane.prepareNotesViewPane(notesViewController);
-            notesViewPane.setGraphReport(notesViewController);
+        final Graph activeGraph = GraphManager.getDefault().getActiveGraph();
+
+        if (activeGraph != null && pluginReport.getGraphReport().getGraphId().equals(activeGraph.getId())) {
+            boolean hasLowLevel = false;
+            for (final String tag : pluginReport.getTags()) {
+                if (LOW_LEVEL_TAG.equals(tag)) {
+                    hasLowLevel = true;
+                    break;
+                }
+            }
+
+            // omit low level plugins which are note useful as notes
+            if ((!pluginReport.getPluginName().contains(NOTES_VIEW_PLUGIN_NAME)) && !hasLowLevel) {
```

Comment:
I think this condition needs to looked at again as I'm getting notes being recorded Notes View read and write state and Quality Control View update state.

I note that the condition previously also did a check for the plugin name containing Quality Control View which has been omitted in the changes

Reply:
Ah yes there was a couple of issues introduced from last minute changes I made. These have been resolved.